src/components/Tree/TreeItem/TreeItem.tsx

```diff
@@ -198,11 +198,12 @@ export const TreeItem = memo(
                     FOCUS_VISIBLE_STYLE,
                     !isActive && !isSelected && 'focus-within:tw-bg-box-neutral',
                     'tw-outline-none tw-ring-inset tw-group tw-px-2.5 tw-no-underline tw-leading-5 tw-h-10',
-                    isSelected
+                    isSelected && !transform?.y
```

Comment:
why use transform.y? Can't you use the isActive flag?

Reply:
No, because we want the active item to have `tw-text`text` color (not white) when an not active item is dragged over displacing the active one.

Otherwhise you'll have this: 
<img width="353" alt="Screenshot 2023-04-28 at 14 34 17" src="https://user-images.githubusercontent.com/357211/235149076-0bc25b60-b8a0-4cd6-8bf9-c99feefee9ff.png">

instead of:
<img width="342" alt="Screenshot 2023-04-28 at 14 40 22" src="https://user-images.githubusercontent.com/357211/235150144-681ac342-bf6b-4880-abad-b00e0931e0db.png">